src/Sfx/App/partials/date-picker.html

```diff
@@ -0,0 +1,14 @@
+<div class="col-md-3">
+    <p class="input-group">
+        <input type="text" class="input-flat clear-disabled" uib-datepicker-popup="{{format}}"
+            ng-model="ngModel" is-open="opened"
+            min-date="minDate" max-date="maxDate" init-date="initDate"
+            datepicker-options="dateOptions" placeholder="{{placeHolderText}}" show-button-bar="false"
+            ng-required ng-readonly="readOnly" ng-click="ctrl.openPopup($event)" />
+        <span class="input-group-btn">
+            <a class="calendar-link" ng-click="ctrl.openPopup($event)" >

```

Comment:
Please add role="button" as it works like a button instead of a link. 
Also, if clicking on it will open a popup, please set aria-haspopup="true"
Please set href="" so that it can capture the focus

Reply:
Sure, will do.